caduceus.yaml

```diff
@@ -230,59 +230,46 @@
   # authHeader provides the list of basic auth headers that caduceus will accept
   # as authorization
   # (Optional)
-  authHeader: ["xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=","xxxxxxxxxxxxxxxxxxxxxx=="]
-
-########################################
-#   Startup Related Configuration
-########################################
-
-  # start is a way to bypass the need to wait for the duration before accepting
-  # events.  The wait is needed to ensure caduceus knows all the hooks it needs
-  # to deliver events to.  Presently the only 2 ways to do that for an SNS
-  # backed service is to:
-  #
-  #     1. Wait until all the hooks have timed out and re-registered (duration)
-  #     2. Ask some server that has the entire list to share it (apiPath)
-  #
-  # TBD add details
-  start:
-    duration: 60000000000
-    apiPath: "https://tr1d1um.example.com:443/api/v2/hooks"
-    sat:
-      path: "https://jwt.example.com/get"
-      id: "magic-id"
-      secret: "xxx"
-      capabilities: "webpa:api:.*:all"
-
-  # waitForDns TBD
-  waitForDns: 0
-
-  # soa is a way for the caduceus machine to check to see if it's DNS name is
-  # present before it tries to use it's DNS name to register with services like
-  # AWS Route53.  This check goes directly to the SOA record authority and
-  # bypasses the various caching servers that could cache the non-existence
-  # record which generally have VERY long TTLs.  Ultimately this way caduceus
-  # can restart much faster.
-  # (Optional)
-  soa:
-    provider: "example.awsdns-19.com:53"
+  authHeader: ["xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=","dXNlcjpwYXNz"]
 
 ########################################
 #   Webhook Related Configuration
 ########################################
+  # Argus Config for storing the webhook information
+  argusConfig:
```

Comment:
Should we make the toplevel field `webhook` here and place all the webhook related configs under it? Then we can do the same for scytale and keep both consistent.